plugins/lookup/tss.py

```diff
@@ -118,15 +118,17 @@
 sdk_is_missing = False
 
 try:
-    from thycotic import __version__ as sdk_version
-    from thycotic.secrets.server import (
-        SecretServer,
-        SecretServerError,
-        PasswordGrantAuthorizer,
-    )
+    from thycotic.secrets.server import SecretServer, SecretServerError
 except ImportError:
     sdk_is_missing = True
```

Comment:
You could also use the technique described in https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/dev_guide/testing/sanity/import.rst to provide this exception to users who ask for it (with increased verbosity).

Reply:
I tried this approach first, then refactored it out, as it'll meant adding a repetitive `if LooseVersion(sdk_version) < LooseVersion('1.0.0')` conditional to the import section. I'm not sure that's appropriate?

Reply:
How is that needed? You just need to store the `ImportError` in a variable to implement this, nothing else.

Reply:
See latest commit.